AppMusic/ViewController.swift

```diff
@@ -6,14 +6,160 @@
 //
 
 import UIKit
+import AVFoundation
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, AVAudioPlayerDelegate {
+    
+    struct Song{
```

Comment:
Sai coding convention
```suggestion
    struct Song {
```